modules/apps/commerce/commerce-address-web/src/main/resources/META-INF/resources/commerce_country/commerce_regions.jsp

```diff
@@ -106,4 +106,27 @@ CommerceRegionsDisplayContext commerceRegionsDisplayContext = (CommerceRegionsDi
 			</liferay-ui:search-container>
 		</aui:form>
 	</div>
+
+	<aui:script>
```

Comment:
Can you try putting this into a separate JS file instead of a new `aui:script` tag?

It does introduce more complexity into the PR, so you might want to split it into 2-3 PRs perhaps

Reply:
Hey @kresimir-coko, do you have a recommended approach? I mean, liferay-frontend:component or should I use like @izaera was doing at https://github.com/liferay-frontend/liferay-portal/pull/2590/commits/db93baac258acee45648246f862b2022dd5a1d29?